packages/providers/src.ts/base-provider.ts

```diff
@@ -12,7 +12,6 @@ import { Deferrable, defineReadOnly, getStatic, resolveProperties } from "@ether
 import { Transaction } from "@ethersproject/transactions";
```

Comment:
It seems that this file contains a lot of formatting changes. Can we revert them?